CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 * Statistics for CRUD operations on router (#224).
+* Integrate CRUD statistics with `metrics` (#224).
```

Comment:
I think that CRUD users may don't know about metrics, and it is better to say that now it is possible to collect statistics about requests to CRUD storages.

Reply:
> it is possible to collect statistics about requests to CRUD storages
But that's not true, there is no distinct metrics for storages. And it does not related to `metrics` integration. But you're right about "CRUD users may don't know about metric", I'll provide a link to repo.

Reply:
Added a link to CHANGELOG